src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -23,7 +23,7 @@ class FormStartControls extends React.Component {
   }
 
   handleLoadPrefill = () => {
-    if (this.props.prefillAvailable) {
+    if (this.props.prefillAvailable && !this.props.noPrefill) {
```

Comment:
Do you want to throw this in the proptypes list?

Reply:
Might be worth documenting the difference between `prefillAvailable` and `noPrefill` somewhere, since they read pretty close. Just one of those things we might not understand a few months down the road.

Reply:
Ahh, great point. Will do. :+1: